dotcom-rendering/src/lib/getBridgetVersion.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react';
+import { getEnvironmentClient } from './bridgetApi';
+
+const isCorrectBridgetVersion = (
+	version: string,
+	requiredVersion: number,
+): boolean => {
+	const majorVersion = parseInt(version);
+	return majorVersion >= requiredVersion;
```

Comment:
I think this is fine assuming we only ever want to compare major versions. If we end up needing to compare minor/patch versions at a later date, it might be good to bring in a semver library to handle parsing and comparison.

Reply:
This is totally fair, I've updated to use the [compare-versions ](https://github.com/omichelsen/compare-versions) library. This will give us a bit more flexibility going forward

Reply:
This makes sense, we've added the compare-versions library to handle this